src/the-map.tsx

```diff
@@ -54,8 +60,22 @@ export function TheMap({ selectedRoutes, showUserLocation }: Props) {
                   isAnimatedMarker ? 'animated-marker' : ''
                 }`,
                 html: `<div class="arrow" style="transform: rotate(${p.direction}deg)"></div>`,
-              })}
-            />
+              })}>
+              <Popup>
+                {
+                  // TODO - look up the tracker ID in the vehicles.csv and extract
+                  //        the "accessibility" flag, to decide whether to show
+                  //        the corresponding icon or not
+                  //      - find a way to determine the current route of this vehicle
+                  //        and display it in a popup
+                }
+                {t('label.board')}
+                {':'} {p.board} {'♿'}
+                <br />
+                {t('label.route')}
+                {': xx'}
```

Comment:
I removed this part for now, because I haven't prepared a data structure for holding info that maps a vehicle to a route; that's my next step and it will be addressed in a separate ticket.